### PR TITLE
Remove find refs window

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

File: src/EditorFeatures/Core/GoToImplementation/GoToImplementationCommandHandler.cs [hunk 1]
```diff
@@ -67,17 +65,12 @@ public void ExecuteCommand(GoToImplementationCommandArgs args, Action nextHandle
...
+            Document document, int caretPosition, IFindUsagesService streamingService)
         {
             var streamingPresenter = GetStreamingPresenter();
 
-            var streamingEnabled = document.Project.Solution.Workspace.Options.GetOption(FeatureOnOffOptions.StreamingGoToImplementation, document.Project.Language);
-            var canUseStreamingWindow = streamingEnabled && streamingService != null;
-            var canUseSynchronousWindow = synchronousService != null;
 
-            if (canUseStreamingWindow || canUseSynchronousWindow)
+            if (streamingService != null)
```
Comment (Pilchie) [inline src/EditorFeatures/Core/GoToImplementation/GoToImplementationCommandHandler.cs]: Is not having this something we should support?

**Comment by CyrusNajmabadi** [comment]
Tagging @amcasey @brettfo This should *not* affect you as you have moved off of the old system.  But i wanted to still make you aware of this change.

**Comment by amcasey** [comment]
AFAIK JS/TS is ready, but let me know if you discover I've missed something. ;)

**Comment by CyrusNajmabadi** [comment]
retest this please

**Comment by CyrusNajmabadi** [comment]
@jasonmalinowski I'm pretty certain TS's changes must have RIed/FIed over to your neck of the woods.  So this should be safe to merge.   I can't actually verify this, so you might want to do some devdiv git-fu to actually ensure the change made it to ml's branch.  :)

Thanks!

**Comment by CyrusNajmabadi** [comment]
@jasonmalinowski Would you be willing to shephard this in?  I know you're a fan of big red PRs.

**Comment by jasonmalinowski** [comment]
@CyrusNajmabadi Sure, I'll take a look at getting it in. Might be a week or two as I'm surrounded by sheep right now.

**Comment by CyrusNajmabadi** [comment]
Sounds good :)   Enjoy the sheep!

**Comment by CyrusNajmabadi** [comment]
Hrmm... That didn't come out like i intended it to...

**Comment by CyrusNajmabadi** [comment]
@sharwell @jasonmalinowski @Pilchie I imagine this can be taken now, as any TS changes must have flowed into your branch by now :)  Do any of you want to Shepard it in?  It is -1k red :)

**Comment by jinujoseph** [comment]
@jasonmalinowski , @CyrusNajmabadi  should we retarget this for post 15.5 branch?

**Comment by jasonmalinowski** [comment]
@jinujoseph Nah, by the time we are ready to merge this master will be open again.
@CyrusNajmabadi Yep, still on the radar. Internal code flow reasons mean we're not _quite_ ready to take yet.

**Comment by CyrusNajmabadi** [comment]
@jasonmalinowski This has been open for 7 months now.  can it please be taken :)

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide @sharwell  in case Jason is too slammed :)

**Comment by CyrusNajmabadi** [comment]
@dotnet-bot retest windows_debug_unit64_prtest please

**Comment by CyrusNajmabadi** [comment]
Thanks!

**Pull request merged**